pinot-common/src/main/java/org/apache/pinot/common/response/BrokerResponse.java

```diff
@@ -76,11 +76,12 @@ default void toOutputStream(OutputStream outputStream)
   /**
    * Returns the processing exceptions encountered during the query execution.
    */
-  List<QueryProcessingException> getExceptions();
+  // TODO: Rename this method
+  List<BrokerResponseErrorMessage> getExceptions();
```

Comment:
This change will cause backward incompatibility for clients. If you search for `QueryProcessingException` in:
* `org:startreedata` - there are multiple places where it is used in startree-pinot. It will help if you have a PR ready to remove references as it is not trivial for others to fix the compilation issue
* All of github - It is used trinodb. https://github.com/trinodb/trino/pull/14399/files

I suggest you deprecate and keep this function as is. Anyway you plan to rename it.

Reply:
This PR will break backward compatibility, that is expected

Reply:
Oh, I see, that is part of the driver. We should not break backward compatibility there.

Reply:
I've renamed the name back to the original one